### PR TITLE
No need for mention of Bitnami

### PR DESCRIPTION
**Pull request by sebgoa**
since we redirect to our OAuth

### CODE REVIEW

**Comment by sebgoa** [comment]
@arapulido Have a look and let me know what you think. When checking the hub live, this struck me. 

It is clear this is a bitnami project, the login redirects to our OAuth page and there is no other Login possibilities, so I don't see the need to mention it.

**Comment by prydonius** [comment]
+1 I was thinking of changing this too

**Comment by prydonius** [comment]
(we need to change this in two more places, I'll update the PR)

File: src/app/app.component.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
       </li>
       <li *ngIf="!loggedIn">
         <a class="reverse" href="/auth">
-          Login with Bitnami
+          Login
```
Comment (arapulido) [inline src/app/app.component.html]: That's fine, but as we are on it, I think it should be Login / Sign up

**Pull request merged**